community/contributing/docs_writing_guidelines.rst

```diff
@@ -0,0 +1,345 @@
+
+**Do** use simple present, preterit or future.
+
+.. highlight:: markdown
+
+    Vector2 move ( Vector2 rel_vec )
+    Moves the body in the vector's direction. The body **stops** if it collides with an obstacle. [...]
+
+You may use the progressive tense to describe actions that are
+continuous in time. Anything like animation or coroutines.
+
+.. tip::
+
+    Verbs can turn into adjectival nouns with -ing. This is not a
+    conjugation, so you may use them: ``the remaining movement``,
+    ``the missing file``, etc.
+
+Remove unnecessary adverbs and adjectives
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Write as few adjectives and adverbs as possible. Only use them if they
+add key information to the description.
+
+**Don't** use redundant or meaningless adverbs. Words that lengthen the
+documentation but don't add any information:
+
+.. highlight:: markdown
+
+    **Basically** a big texture [...]
+
+**Do** write short sentences in a simple, descriptive language:
+
+.. highlight:: markdown
+
+    A big texture [...]
+
+Ban these 8 words
+~~~~~~~~~~~~~~~~~
+
+**Don't** ever use these 8 banned words:
+
+1. obvious
+2. simple
+3. basic
+4. easy
+5. actual
+6. just
+7. clear
+8. however (some uses)
+
+Game creation and programming aren't simple, and nothing's easy to
+someone learning to use the API for the first time. Other words in the
+list, like ``just`` or ``actual`` won't add any info to the sentence.
+Don't use corresponding adverbs either: obviously, simply, basically,
+easily, actually, clearly.
+
+**Don't** example. The banned words lengthen the description and take
+attention away from the most important info:
+
+.. highlight:: markdown
+
+    **TextureRect**
+    Control frame that **simply** draws an assigned texture. It can stretch or not. It's a **simple** way to **just** show an image in a UI.
+
+**Do** remove them:
+
+.. highlight:: markdown
+
+    **TextureRect**
+    [Control] node that displays a texture. The texture can stretch to the node's bounding box or stay in the center. Useful to display sprites in your UIs.
+
+"Simple" never helps. Remember, for other users, anything could be
+complex or frustrate them. There's nothing like a good old *it's simple*
+to make you cringe. Here's the old brief description, the first sentence
+on the Timer node's page:
+
+.. highlight:: markdown
+
+    **Timer**
+    A **simple** Timer node.
+
+**Do** explain what the node does instead:
+
+.. highlight:: markdown
+
+    **Timer**
+    Calls a function of your choice after a certain duration.
+
+**Don't** use "basic", it is too vague:
+
+.. highlight:: markdown
+
+    **Vector3**
+    Vector class, which performs **basic** 3D vector math operations.
+
+**Do** use the brief description to offer an overview of the node:
+
+.. highlight:: markdown
+
+    **Vector3**
+    Provides essential math functions to manipulate 3d vectors: cross product, normalize, rotate, etc.
```

Comment:
3D ?

Reply:
Ah gotta up the case for all 2d/3d, thanks, forgot that